### DISCUSSION TITLE
If condition inside another if condition

### DISCUSSION THREAD

**Discussion by fervaldies** [idaholab/moose#28542] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hello, I would like to know why this expression is not working. I would like the function to be 50, 100, 150, 200 and 300, for the different timings of 1, 2, 3, 4 and more, respectively.

  [./path_x]
    type = ParsedFunction
    expression = 'if(t<1,50,if(t<2,100,if(t<3,150,if(t<4,200,300))))'
  [../]

**Comment by GiudGiud** [comment]
Hello

Are you checking at t=1 exactly? Since you used < it would have jumped already to the next value

Expression looks correct to me

**Reply by fervaldies** [reply, marked as answer]
Hello, my apologies, it is correct and it is working, my error was in "path_z" and that is why I could not see the behaviour that I wanted. Thanks! :)
